src/components/v5/common/Table/Table.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from '@tanstack/react-table';
+import clsx from 'clsx';
+import { TableProps } from './types';
+import { useMobile } from '~hooks';
+
+const displayName = 'v5.common.Table';
+
+const Table = <T,>({
+  className,
+  tableTitle,
+  columns,
+  fields,
+  burgerColumn,
```

Comment:
If this is generic, then you shouldn't add such props here, same as `setSelectedRowId`. You can later create another component that will use this base table and will have the actions column added by default.

Reply:
I've created another component like <TableWithBurger /> which uses  this base table and the actions column added by default.

Reply:
Then you shouldn't include such props here